src/core/display/Container.js

```diff
@@ -606,17 +606,23 @@ Container.prototype.renderCanvas = function (renderer)
 
 /**
  * Destroys the container
- * @param [destroyChildren=false] {boolean} if set to true, all the children will have their destroy method called as well
+ * @param [opts] {object} Options parameter
+ * @param [opts.children=false] {boolean} if set to true, all the children will have their destroy
+ *      method called as well. 'opts' will be passed on to those calls.
+ *
  */
-Container.prototype.destroy = function (destroyChildren)
+Container.prototype.destroy = function (opts)
 {
+    opts = opts || {};
```

Comment:
@marekventur I think we should add a boolean as an alternative function signature and _not_ deprecate this functionality.  There is an open question of whether passing `true` into `destroy()` would be a shortcut for `{children: true}` or `{children: true, texture: true, baseTexture: true}`. I'm in favor of the latter because it doesn't break current implementations and seems more convenient. Thoughts?